shepherd/shepherd.py

```diff
@@ -0,0 +1,438 @@
+from multiprocessing import Process
```

Comment:
Do you really want to commit this file? It seems to be a concatenation of all the others.

Reply:
No. Thanks for pointing it out. I will remove all unncessary files and code update the PR soon.